Library/Homebrew/utils/github/actions.rb

```diff
@@ -85,5 +104,9 @@ def relevant?
         @file.descend.next.to_s != ".."
       end
     end
+
+    # Generic GitHub Actions error.
+    class Error < RuntimeError
+    end
```

Comment:
Are you planning to use this?

Reply:
Yes, I used it here:
```
if name.include?(delimiter) || value.include?(delimiter)
  raise Error, "`name` and `value` must not contain the delimiter"
end
```

Reply:
Sorry, I meant: do you need to `rescue` this anywhere? Otherwise you could just do
```ruby
  raise "`name` and `value` must not contain the delimiter"
```

Reply:
Ah, I guess I could've just done that 🤦🏻‍♂️

Reply:
Not too late to change it!